scripts/website.js

```diff
@@ -600,3 +617,12 @@ if (isMain) {
     console.log('Done Processing');
   })();
 }
+
+// Modified from github-slugger
+function createSlug(value) {
+  if (typeof value !== 'string') {
+    return '';
+  }
+  value = value.toLowerCase();
+  return value.replace(/[^a-z0-9-_\s]/, '').replace(/ /g, '-');
+}
```

Comment:
Yeah default slugger was removed in marked@9 apparently. Recommendation is to switch to github-slugger npm package. See discussion from this issue: https://github.com/markedjs/marked/issues/2983.

I copied most of the core logic from github-slugger into this function. github-slugger is more sophisticated, but I wanted to inline our slugging logic because I don't want to have to go through every single header every time we have to upgrade github-slugger.